src/api/app/jobs/project_do_project_release_job.rb

```diff
@@ -0,0 +1,10 @@
+class ProjectDoProjectReleaseJob < ApplicationJob
+  queue_as :quick
+
+  def perform(project_id, params)
+    ProjectDoProjectReleaseJob.perform_later(
```

Comment:
Arghh I copied bjoern's [fix](https://github.com/evanrolfe/open-build-service/pull/6/files) into the wrong place. I will sort this out tomorrow morning, it seems my brain is getting tired...